scripts/local/docker-compose.yml

```diff
@@ -0,0 +1,148 @@
+# This is the docker-compose script for local development.
+# For the latest version of the docker images see:
+# https://github.com/Azure/pcs-cli/blob/master/remotemonitoring/single-vm/docker-compose.dotnet.yml
+#
+# Usage:
+# - install Docker Compose, see https://docs.docker.com/compose/install
```

Comment:
btw, doesn't docker compose just come with a docker install?  i.e. IIRC you don't install Docker compose, you install Docker.

Reply:
It depends. Windows and Mac by default have it installed when you install docker. However, Linux machines need to install docker compose separately.